tests/page-objects/standard/enketo/delivery.wdio.page.js

```diff
@@ -1,6 +1,7 @@
-const OUTCOME = {liveBirth: 'healthy', stillBirth: 'still_birth', miscarriage: 'miscarriage'};
-const LOCATION = {facility: 'f', homeAttendant: 's', homeNoAttendant: 'ns'};
-
+const OUTCOME = { liveBirth: 'healthy', stillBirth: 'still_birth', miscarriage: 'miscarriage' };
+const LOCATION = { facility: 'f', homeAttendant: 's', homeNoAttendant: 'ns' };
+const MINIMUM_BROWSER_VERSION = '90';
```

Comment:
Instead of copying this value which will make life harder when we bump it, let's expose a common function somewhere for `isTestingLegacyChrome` or something?